spring-security-modules/spring-security-mvc-boot-2/src/main/resources/spring-security-login-redirect.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
not needed anymore right?

Reply:
Not sure, I didn't find any information related to this fact.

Reply:
I meant the file, not the line. the file is not being used as far as I know, right?

Reply:
It contains the XML configuration that I presented in the article. 

Also, it is related to the line
_//@ImportResource({"classpath*:spring-security-login-redirect.xml"})_,
from the _LoginRedirectApplication_.

I intentionally kept that line and file in case someone would like to run the application with the XML configuration.